xml/article_sap_trento.xml

```diff
@@ -745,95 +979,124 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
             <title><guimenu>Cluster settings</guimenu></title>
             <listitem>
-              <para><guimenu>Connection settings</guimenu>: set
-                the Trento administrator that is used to connect
-                to your hosts. In this article, it is
-                <systemitem class="username"
-                  >&trentoadmin;</systemitem>. </para>
+              <remark>toms 2022-04-28: review #2, change 2</remark>
+              <para><guimenu>Connection settings</guimenu> tab: 
+                set the user that the &t.server; will use to connect
+                via SSH to the cluster nodes to perform the HA
+                configuration checks.
+                In this article, it is <systemitem class="username"
+                  >&trentoadmin;</systemitem>.
+              </para>
+              <itemizedlist>
+                <listitem>
+                  <para>The default connection user in Azure environments
+                    is the Azure admin user used to deploy the VMs.
+                  </para>
+                </listitem>
+                <listitem>
+                  <para>The default connection user in AWS and GCP is &rootuser;.</para>
+                </listitem>
+              </itemizedlist>
+              <para>
+                If you change the default connection user, you need to click on
+                <guimenu>Apply Connection Settings</guimenu> to save the changes.
+              </para>
+              <informalfigure>
+                <mediaobject>
+                  <imageobject>
+                    <imagedata fileref="trento-web-cluster-settings-hana_cluster.png" width="70%"/>
+                  </imageobject>
+                  
+                </mediaobject>
+              </informalfigure>
             </listitem>
             <listitem>
               <para>
-                <guimenu>Checks selection</guimenu>:
-                select/deselect a complete category of a test
-                (such as &corosync;, &pace; etc.) or individual
-                tests. </para>
+                <guimenu>Checks selection</guimenu> tab:
+                select/deselect a complete category of checks
+                (such as &corosync; or &pace; etc.) or individual
+                checks. Whenever you make a new selection of checks you need to
+                click <guimenu>Select Checks for Execution</guimenu> to save the change.
+              </para>
+              <!-- still change 2 -->
             </listitem>
           </itemizedlist>
-          <para> Do not forget to save your changes before closing the
-            window. </para>
+          <para> <remark> alberto: from review #2, change 2 </remark> Whenever a change is made and saved in either tab, if any check is selected a button will show up allowing the user to start the check execution right away: <remark> alberto: include last screenshot in review #2, change 2 </remark> </para>
```

Comment:
Please add the following screenshot after the paragraph:
![image](https://user-images.githubusercontent.com/91731013/165967228-ead32be0-a59a-426f-98ab-83a5af1947a4.png)

Reply:
Thanks. Actually when I went through it, I didn't include it because in comparison to the previous screenshot for me it was a minor change (just the button). That wouldn't qualify it for me to include it. Furthermore, as the target audience for this guide are SAP experts, I think they don't miss this screenshot. We can include some text, if needed.

Additionally, keep in mind, we need to update all screenshots. Whenever the team decides to change the wording or the placement we need to update it.

I didn't miss it, but decided not to include it for the above reasons. If you still think this screenshot has some value, I can include it of course.

Reply:
Fair enough! In that case, please remove the ":" at the end of the paragraph.